check/check.go

```diff
@@ -105,18 +106,21 @@ func (c *Check) run() State {
 	// without tests return a 'WARN' to alert
 	// the user that this check needs attention
 	if c.Scored && len(strings.TrimSpace(c.Type)) == 0 && c.Tests == nil {
+		c.WarnReason += "There is no tests"
```

Comment:
```suggestion
		c.WarnReason += "There are no tests"
```

Also, does this need to be `+=` or could it just be a normal assignment with `=`? If `+=` what are we appending this to?

Reply:
Can you comment on why it's += and not just regular = here?